### ISSUE TITLE
This error

### ISSUE THREAD

**Issue by dyedgreen** [Adezandee/fast-eslint#32]
The validator seems to fail when trying to validate code with a 'this' expression.

**Comment by Adezandee** [comment]
Hi @dyedgreen, can you please be more specific about your problem and provide your `.eslintrc` config file along side a code sample that trigger the error. Thanks

**Comment by dyedgreen** [comment]
Thanks for the fast reply @Adezandee :)

This is the .eslintrc file
``` json
// Use this file as a starting point for your project's .eslintrc.
// Copy this file, and add rule overrides as needed.
{
  "extends": "airbnb"
}
```

However the problem went away after restarting the validator. It might have been just something random or it might have been a very specific problem with the following code (due to missing the `super()` statement perhaps?)
``` javascript
'use strict';

class SomeClass extends SomeOtherClass {
  constructor(someOption) {
    // No super() call here!
    this._someOption = someOption === undefined ? false : someOption;
  }

  // Some function
  function() {
    setInterval(() => {
      // Here is where the error occurred
      this._someOption = true;
    }, 1000);
  }
}
```

**Comment by Adezandee** [comment]
@dyedgreen indeed after updating `fast-eslint` or updating your `.eslintrc`, if you experience unexpected errors try to restart atom. This solve most of conflicting memory loaded/updated code with `eslint`.
